modules/network-observability-without-loki.adoc

```diff
@@ -4,16 +4,24 @@
 :_mod-docs-content-type: REFERENCE
 [id="network-observability-without-loki_{context}"]
 = Network Observability without Loki
-You can use Network Observability without Loki by not performing the Loki installation steps and skipping directly to "Installing the Network Observability Operator". If you only want to export flows to a Kafka consumer or IPFIX collector, or you only need dashboard metrics, then you do not need to install Loki or provide storage for Loki.  Without Loki, there is no *Network Traffic* panel under *Observe*, which means there is no overview charts, flow table, or topology. The following table compares available features with and without Loki:
+You can use Network Observability without Loki by not performing the Loki installation steps and skipping directly to "Installing the Network Observability Operator". If you only want to export flows to a Kafka consumer or IPFIX collector, or you only need dashboard metrics, then you do not need to install Loki or provide storage for Loki.  Without Loki, there is no *Network Traffic* panel under *Observe*, which means there is no overview charts, flow table, or topology. The following table compares available features with and without Loki.
+
+[IMPORTANT]
```

Comment:
what about adding that directly in the table? E.g.

- "Complete filtering and aggregations capabilities (such as per pod)" => With Loki YES, Without NO
- "Partial filtering and aggregations capabilities (such as per workload or namespace)" => With Loki YES, Without YES

Reply:
(not sure if I use the best wording, but you get the idea)

Reply:
Okay I like the idea of reducing some of this example text with tabular info instead.